cmd/plugin.go

```diff
@@ -133,6 +133,7 @@ func stashPluginMain(args []string) {
 			fmt.Println("PluginType = \"FileTransfer\"")
 			fmt.Println("ProtocolVersion = 2")
 			fmt.Println("SupportedMethods = \"stash, osdf\"")
+			fmt.Println("StartdAttrs = \"PluginVersion\"")
```

Comment:
Does this get copied literally into the startd ad?  Or does it get a prefix based on the plugin name somehow?

Given there are multiple plugins at play, I think we'll want some prefix (`PelicanPluginVersion`).